samples/bluetooth/alexa_gadget/src/gadgets/CMakeLists.txt

```
@@ -0,0 +1,181 @@
+    ${ALEXA_EMBEDDED_PATH}/AlexaGadgetsProtobuf/Alexa.Gadget.SpeechData/Speechmarks/alexaGadgetSpeechDataSpeechmarksDirective.proto
+    ${ALEXA_EMBEDDED_PATH}/AlexaGadgetsProtobuf/Alexa.Gadget.SpeechData/Speechmarks/alexaGadgetSpeechDataSpeechmarksDirectivePayload.proto
+    ${ALEXA_EMBEDDED_PATH}/AlexaGadgetsProtobuf/Alexa.Gadget.MusicData/Tempo/alexaGadgetMusicDataTempoDirective.proto
+    ${ALEXA_EMBEDDED_PATH}/AlexaGadgetsProtobuf/Alexa.Gadget.MusicData/Tempo/alexaGadgetMusicDataTempoDirectivePayload.proto
+    ${CMAKE_CURRENT_SOURCE_DIR}/proto/custom_event.proto
+)
+
+# Populate list of expected .c files output from protobuf generation
+foreach(file ${PROTO_FILES})
+    get_filename_component(FILE_PATH ${file} DIRECTORY)
+    get_filename_component(FILE_BASE ${file} NAME_WE)
+    list(APPEND PROTOC_INC_DIRS "-I${FILE_PATH}")
+    list(APPEND PROTO_BYPRODUCTS "${PROTO_COMPILATION_DIR}/${FILE_BASE}.pb.c")
+endforeach()
+
+# Populate list of compilation commands for protoc compiler.
+foreach(file ${PROTO_FILES})
+    get_filename_component(FILE_PATH ${file} DIRECTORY)
+    get_filename_component(FILE_NAME ${file} NAME)
+    set(PROTOC_ARGS "${PROTOC_INC_DIRS}")
+    list(APPEND PROTOC_ARGS "--nanopb_out=${PROTO_COMPILATION_DIR}")
+    list(APPEND PROTOC_ARGS "${FILE_NAME}")
+    list(APPEND PROTOC_CMD_LIST COMMAND ${CMAKE_COMMAND} -E chdir ${FILE_PATH} ${PROTOC_BIN} ${PROTOC_ARGS})
+endforeach()
+
+# Add custom target to compile all .proto files
+add_custom_target(
+    protobuf_generation
+    COMMAND ${PROTOC_CMD_LIST}
+    BYPRODUCTS ${PROTO_BYPRODUCTS}
+)
+
+zephyr_library_named(ALEXA_GADGETS_PROFILE)
+
+zephyr_include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+zephyr_include_directories(${PROTO_COMPILATION_DIR})
+zephyr_include_directories(${NANOPB_PATH})
+
+zephyr_library_sources_ifdef(CONFIG_GADGETS_PROFILE_ENABLE
+    ${CMAKE_CURRENT_SOURCE_DIR}/gadgets_profile.c
+    ${NANOPB_PATH}/pb_common.c
+    ${NANOPB_PATH}/pb_encode.c
+    ${NANOPB_PATH}/pb_decode.c
+    ${PROTO_COMPILATION_DIR}/accessories.pb.c
+    ${PROTO_COMPILATION_DIR}/common.pb.c
+    ${PROTO_COMPILATION_DIR}/device.pb.c
+    ${PROTO_COMPILATION_DIR}/firmware.pb.c
+    ${PROTO_COMPILATION_DIR}/directiveParser.pb.c
+    ${PROTO_COMPILATION_DIR}/directiveHeader.pb.c
+    ${PROTO_COMPILATION_DIR}/eventParser.pb.c
+    ${PROTO_COMPILATION_DIR}/eventHeader.pb.c
+    ${PROTO_COMPILATION_DIR}/alexaDiscoveryDiscoverDirective.pb.c
+    ${PROTO_COMPILATION_DIR}/alexaDiscoveryDiscoverResponseEvent.pb.c
+    ${PROTO_COMPILATION_DIR}/alexaDiscoveryDiscoverResponseEventPayload.pb.c)
+
+zephyr_library_sources_ifdef(CONFIG_GADGETS_CAPABILITY_NOTIFICATIONS_ENABLE
+    ${PROTO_COMPILATION_DIR}/notificationsSetIndicatorDirective.pb.c
+    ${PROTO_COMPILATION_DIR}/notificationsSetIndicatorDirectivePayload.pb.c
+    ${PROTO_COMPILATION_DIR}/notificationsClearIndicatorDirective.pb.c
+    ${PROTO_COMPILATION_DIR}/notificationsClearIndicatorDirectivePayload.pb.c)
```

Could you place the end `)` independently, as you do in other multiline functions in this file ?
```suggestion
    ${PROTO_COMPILATION_DIR}/notificationsClearIndicatorDirectivePayload.pb.c
)
```

Same goes for following lines.

---

Fixed